client/verta/verta/deployment/_deployedmodel.py

```diff
@@ -116,6 +124,8 @@ def from_url(cls, url, token):
         token : str or None
             Prediction token. Can be copy and pasted directly from the Verta Web App. If the deployment
             does not use a token, ``None`` should be passed in as the argument.
```

Comment:
`token` originally wasn't optional per se; rather, `None` was meant to be passed explicitly for endpoints that have token-based permissions disabled. But I'm realizing that making this optional might be functionally equivalent anyway 🤔 
```suggestion
        token : str, optional
            Prediction token. Can be copy and pasted directly from the Verta Web App.
```